.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
```

Comment:
Should this file also be in the config folder, (And config_list.txt)?

Reply:
yes it should, nice catch 😊 added